Source/WebCore/page/LocalFrameView.cpp

```diff
@@ -6188,6 +6193,43 @@ void LocalFrameView::scrollbarWidthChanged(ScrollbarWidth width)
     m_needsDeferredScrollbarsUpdate = true;
 }
 
+int LocalFrameView::scrollbarGutterWidth(bool isHorizontalWritingMode) const
+{
+    if (verticalScrollbar() && verticalScrollbar()->isOverlayScrollbar())
+        return 0;
+
+    if (!verticalScrollbar() && !(scrollbarGutterStyle().isAuto || ScrollbarTheme::theme().usesOverlayScrollbars()) && isHorizontalWritingMode)
+        return ScrollbarTheme::theme().scrollbarThickness(scrollbarWidthStyle());
+
+    if (!verticalScrollbar())
+        return 0;
+
+    return verticalScrollbar()->width();
+}
+
+
+IntSize LocalFrameView::computeScrollbarGutter() const
+{
+    IntSize scrollbarGutter = { horizontalScrollbarIntrusion(), verticalScrollbarIntrusion() };
+
+    if (isHorizontalWritingMode()) {
+        if (scrollbarGutterStyle().bothEdges)
+            scrollbarGutter.setWidth(scrollbarGutterWidth() * 2);
+        else
+            scrollbarGutter.setWidth(scrollbarGutterWidth());
+    }
+    return scrollbarGutter;
+}
+
+int LocalFrameView::horizontalScrollbarInset() const
```

Comment:
Easily mis-interpreted as "inset related to the horizontal scrollbar". Maybe `insetForLeftScrollbar()`

Reply:
Fixed.